src/main/java/org/folio/bulkops/service/ElectronicAccessService.java

```diff
@@ -48,9 +48,9 @@ public ElectronicAccess restoreItemElectronicAccessItem(String access) {
   }
 
   private String electronicAccessToString(ElectronicAccess access, String delimiter) {
-    var tenantId = isNull(access.getTenantId()) ? folioExecutionContext.getTenantId() : access.getTenantId();
+    log.info("electronicAccessToString: {}, {}", access.getRelationshipId(), folioExecutionContext.getTenantId());
```

Comment:
Remove this logging or make it debug if you really need it.

Reply:
Changed to debug.